### PR TITLE
Browser test and throws()

### PR DESCRIPTION
**Pull request by hakanols**
I used Teston to test es6 code in both  browser and node. Also added throws(). This needed some modification. Added a HTML file that run the tests. The link to run the test in README.md need to be pointed to your projects and GitHub pages pages activated. The version also need to be updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teston",
-  "version": "0.7.0",
+  "version": "0.0.0",
```
Comment (pakastin) [inline package.json]: Hmm.. why version `0.0.0`? 😄

File: README.md [hunk 2]
```diff
@@ -106,6 +106,9 @@ _alias: t.deepEquals_
 
 Assert that `a` and `b` are deeply equal.
 
+## Test
+* [Runt tests in browser](https://hakanols.github.io/teston/test/index.html)
```
Comment (pakastin) [inline README.md]: Could this be https://testonjs.github.io/teston/test/index.html, I can activate it

**Review by pakastin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hakanols) [reply package.json]: Sorry! I panicked. The plan from the beginning was not to push and I did not want to interfere with your versioning. I have pushed a new commit changing the version to 0.8.0.
I have some problem with GitHub. Do not know if my new commits is in this pull request.

File: README.md (hunk 2 above)
Comment (hakanols) [reply README.md]: It should be changed in the last commit. As you writer, you have to activate pages under the repository settings.

**Comment by hakanols** [comment]
Hello!
Is the pull request rejected? I think I have handled  your change requests. Should I Close and do a new one?
BR Håkan

**Comment by pakastin** [comment]
Sorry, I missed this – merged now, thank you so much!

**Comment by hakanols** [comment]
Thank you! You ave really made an impressive amount of repos.

**Comment by hakanols** [comment]
You have to activate GitHub pages for "Runt tests in browser" to work. The link is correct i think.

**Comment by pakastin** [comment]
Yeah, and getting the new github security updates from all those repos makes my inbox scream 😄

**Comment by hakanols** [comment]
Tried to run testes in browser under:
https://github.com/testonjs/teston#test
It fails. Is GitHub pages inactivated by design? Might the link be wrong after all?

BTW: I have deleted my fork

**Comment by pakastin** [comment]
Should work now. My bad, forgot to activate pages, sorry 😬

**Comment by hakanols** [comment]
Excellent! Tested and it is up and running!

**Pull request merged**